src/OpenLoco/src/Windows/Construction/OverheadTab.cpp

```diff
@@ -121,7 +121,7 @@ namespace OpenLoco::Ui::Windows::Construction::Overhead
     // 0x0049ECD1
     static void onUpdate(Window& self)
     {
-        Common::onUpdate(&self, (1 << 5));
+        Common::onUpdate(&self, GhostFlags::flag4);
```

Comment:
Is this a mistake? Looks like it should've been flag5?

Reply:
It was a mistake in the old code.